command/sub8_missions/missions/__init__.py

```diff
@@ -1,8 +1,8 @@
 # flake8: noqa
-from os.path import dirname, basename, isfile
-import glob
-modules = glob.glob(dirname(__file__)+"/*.py")
-__all__ = [ basename(f)[:-3] for f in modules if isfile(f)]
-for name in __all__:
-    if name.startswith('__'):
-        __all__.remove(name)
+import os
+for module in os.listdir(os.path.dirname(__file__)):
+    if module == '__init__.py' or module[-3:] != '.py':
+        continue
+    __import__(module[:-3], locals(), globals())
+del module
+del os
```

Comment:
It finds all of the files in that directory and includes them in the module (Instead of you having to do import x; import y; import z